zOSMF-Samples/workflow_sample_dataset.xml

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/********************************************************************************/
+/* Copyright 2019 IBM Corp.                                                     */
+/*                                                                              */
+/* Licensed under the Apache License, Version 2.0 (the "License");              */
+/* you may not use this file except in compliance with the License.             */
+/* You may obtain a copy of the License at                                      */
+/*                                                                              */
+/*    http://www.apache.org/licenses/LICENSE-2.0                                */
+/*                                                                              */
+/* Unless required by applicable law or agreed to in writing, software          */
+/* distributed under the License is distributed on an "AS IS" BASIS,            */
+/* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.     */
+/* See the License for the specific language governing permissions and          */
+/* limitations under the License.                                               */
+/********************************************************************************/
+-->
+
+<workflow>
+    <workflowInfo>
+        <workflowID>workflow.sample.dataset</workflowID>
+		<workflowDefaultName>Sample workflow to manipulate data sets</workflowDefaultName>
+        <workflowDescription>Sample that demonstrates working with a dataset and gether feedback for a step.</workflowDescription>
```

Comment:
<workflowDescription>Sample that demonstrates working with a dataset and gather feedback for a step.</workflowDescription>